nested/storageAccount.json

```diff
@@ -93,4 +93,4 @@
             "type": "string"
         }
     }
-}
```

Comment:
No change to this file yea?

Reply:
you are right, no intended change in this file.
Fixed it.